### DISCUSSION TITLE
Only in DEV

### DISCUSSION THREAD

**Discussion by ta-soft** [liuweiGL/vite-plugin-mkcert#95] [General]
Hi there,

I only want to use this plugin for my dev. PROD is  already behind nginx with its ssl cert, no need for another certificate. How can I achieve this?

Thank you!

**Comment by liuweiGL** [comment]
https://cn.vite.dev/config/

```ts
export default defineConfig(({ command, mode, isSsrBuild, isPreview }) => {
    return {
         plugins: [
           mode === 'development' && mkcert() 
         ]
    }
})
```
